timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/dataset/DataSetFactoryTest.java

```diff
@@ -123,8 +127,8 @@ public void removeDataSetRemovesTheDataSetFromDisk() throws Exception {
   }
 
   @Test
-  public void removeDataSetRemovesTheDataSetFromTheIndex() throws Exception {
-    dataSetFactory.createImportManager("user", "dataSet");
+  public void deleteDataSetRemovesTheDataSetFromTheIndex() throws Exception {
```

Comment:
For me it is more logical if the test starts with the name of the method it is testing.